pkg/kademlia/routing.go

```diff
@@ -89,38 +89,21 @@ func (rt *RoutingTable) CacheSize() int {
 	return rt.rcBucketSize
 }
 
-// GetBucket retrieves the corresponding kbucket from node id
+// GetNodes retrieves the corresponding kbucket from node id
```

Comment:
maybe update the comment here to say "retrieves nodes within the same kbucket as the given node id"?

Reply:
yes. Made the change